packages/theme/nuxt.config.js

```diff
@@ -200,6 +200,8 @@ export default {
         path: '/my-account/:pageName/:id?',
         component: resolve(__dirname, 'pages/MyAccount.vue')
       });
+      const index = routes.findIndex(route => route.path === "/Product");
```

Comment:
I'm not sure about this one, it looks a bit hacky and I have a feeling that this will have unintended consequences at some point. What if we removed the routes that VSF comes with OOB and defined our own routes explicitly? It would be more readable then.

Reply:
Thanks for the feedback @rafalcymerys . Would something like my updated commit be preferable?

Reply:
Perfect :)